src/components/Preferences/ScheduleSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import ScheduleSelector from "react-schedule-selector";
+
+type PropsType = { color: string; semester: string };
+class Timetable extends React.Component<PropsType> {
```

Comment:
Ideally should be a functional component like the others but should get this in for the demo so we can do it later